views/form.html

```diff
@@ -84,9 +84,18 @@ <h1 class="display-4">Onboarding Process form</h1>
                         </check>
                     </div>
                     <div class="form-group">
-                        <label class="font-weight-bold">Bio</label><em>(optional)</em>
-                        <textarea class="form-control input clearText" name="bio"> {{ @bioText }}</textarea>
-                        <span class="err"> {{ @errors['check'] }}</span>
+                        <label class="font-weight-bold">Bio</label>
+                        <check if='{{ @bioText !="" }}'>
+                            <true>
+                                <textarea class="form-control input clearText" name="bio"> {{ @bioText }}</textarea>
+                                <span class="err"> {{ @errors['check'] }}</span>
+                            </true>
+                            <false>
+                                <span><em>(Not submitted yet)</em></span>
```

Comment:
Will this span be hidden if Bio has been submitted?

Reply:
yes it will be hidden